### ISSUE TITLE
Setup CI

### ISSUE THREAD

**Issue by ielis** [TheJacksonLaboratory/PhenopacketLab#9]
Hi @belkassaby , now that the build passes, could you please setup CI in a similar way you set it up in Biodownloader?

I am wondering though if we should create a separate profile for `node` setup in `frontend` module. The build of that module is taking ages to complete..

**Comment by gerring** [comment]
I would be willing to do this but I am not part of https://github.com/TheJacksonLaboratory organization on github, therefore cannot link in CI.

**Comment by ielis** [comment]
I am not aware that adding a CI requires you or anybody else to be part of any organization. The CI I have on my mind should execute the Maven build and run tests.
Am I missing something? Is there frontend specific thing that needs the contributor organization? Or is this related to privileges? I can grant you Write privileges if you'd like to have them, and you always can open a PR from your fork.

**Comment by belkassaby** [comment]
@ielis yes we should have a CI/CD for the frontend and another one for the backend.

**Comment by belkassaby** [comment]
@gerring ask RIT I think so they can add you to the Jacksonlaboratory organization on github. That's what I did about a year ago.

**Comment by ielis** [comment]
@belkassaby , I'll setup Github actions workflow for running tests just on backend modules.
